draft-ietf-mmusic-mux-exclusive.xml

```diff
@@ -135,11 +135,15 @@
             media associated with the SDP media description ("m=" line).
         </t>
         <t>
-            In an SDP answer, the 'rtcp-mux-only' attribute indicates that the answerer
+            In an SDP answer, the 'rtcp-mux' attribute <xref target="RFC5761"
+            pageno="false" format="default"/>indicates that the answerer
             supports, and accepts usage of, RTP/RTCP multiplexing for the RTP-based media
             associated with the SDP media description ("m=" line).
         </t>
         <t>
+            The usage of the 'rtcp-mux-only' attribue in an SDP answer is unspecified.
```

Comment:
attribute.

I would say "forbidden". "unspecified" is a source of trouble. And I note you forbid it later.